README.md

```diff
@@ -88,7 +88,10 @@ const saml = new SAML(options);
 ```
 
 - **InResponseTo Validation**
-- `validateInResponseTo`: if truthy, then InResponseTo will be validated from incoming SAML responses
+- `validateInResponseTo`:
+  - if `true`, `"always"`, then InResponseTo will be validated from incoming SAML responses
```

Comment:
After your latest changes, this documentation needs to be updated to remove the option to use a boolean.

Reply:
Thanks, you're right, I forgot it